test/OpenTelemetry.Exporter.Zipkin.Tests/ZipkinExporterTests.cs

```diff
@@ -135,7 +135,6 @@ public void SuppresssesInstrumentation()
         [Theory]
         [InlineData(true, false, false)]
         [InlineData(false, false, false)]
-        [InlineData(false, true, false)]
```

Comment:
Currently in our Zipkin tests, only one passed parameter would instantiate a Resource and checks for its tags populating. I left the If(UseTestResource) clause in the code in case we would like to return to the old resource tag checking, but if it makes more sense I can remove that parameter entirely.

Reply:
I think we need to put this test back in since it is also validating serviceName selection from resource.

Just remove the tag validation part:
```csharp
            if (useTestResource)
            {
                serviceName = "MyService";

                exporter.SetLocalEndpointFromResource(ResourceBuilder.CreateEmpty().AddAttributes(new Dictionary<string, object>
                {
                    [ResourceSemanticConventions.AttributeServiceName] = serviceName,
                    ["service.tag"] = "hello world",
                }).Build());
            }
```
* Removed `resoureTags = "\"service.tag\":\"hello world\",";`
* By leaving in `["service.tag"] = "hello world",` the test will now validate tags are NOT included, which is the new behavior.